frontend/src/pages/Home.tsx

```diff
@@ -1,16 +1,25 @@
-import useAllProducts from "../hooks/useAllProducts";
+import useProducts from "../hooks/useProducts";
 import ProductContainer from "../components/ProductContainer";
-import React from "react";
 import LogoutButton from "../components/LogoutButton";
+import {useParams} from "react-router-dom";
 
 export default function Home() {
-    const[products] = useAllProducts();
+    const {excuted} = useParams();
+
+
+let areExcuted : boolean = excuted ? Boolean(excuted) : false ;
```

Comment:
Ich glaub ich versteh einfach nicht, was dieser Parameter `excuted` bewirken soll. Magst du mir das vielleicht kurz erklären?

Reply:
da benuze ich es als boolean so entweder true oder false be areExcuted variable